tasks/photoprism.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: PhotoPrism - create config directory
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: 0777
+  with_items:
+    - "{{ photoprism_config_directory }}/storage"
+
+- name: PhotoPrism - install Docker container
+  docker_container:
+    name: photoprism
+    image: photoprism/photoprism
+    pull: true
+    ports:
+      - "{{ photoprism_port }}:2342"
+    volumes:
+      - "/etc/localtime:/etc/localtime:ro"
+      - "{{ photoprism_config_directory }}/storage:/photoprism/storage:rw"
```

Comment:
Please rename `photoprism_config_directory` to `photoprism_data_directory` to reflect what's stored in the directory